src/components/Books.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import progress from '../progress.svg';
+import Form from './Form';
+
```

Comment:
- [ ] As it is a requirement for this project and best practice, it will be nice to split this file into example `Booklist`, `Book`, `CreateNewBook` as this will make it easier for you to manage your code as it will be more readable.  See the snapshot below.

![snapshot](https://user-images.githubusercontent.com/65852621/146494748-11e3e673-b831-478e-9448-86bf7e65cdd6.png)